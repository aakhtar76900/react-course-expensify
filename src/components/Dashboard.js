import React from 'react';
import {Link} from 'react-router-dom';
import ExpenseList from '../components/ExpenseList';
import ExpenseListFilter from '../components/ExpenseListFilter';

const Dashboard = () => {
    return (
        <div>
            <h3>Expenses Dashboard</h3>
            {/* <Link to="/edit/1"> Item 1 </Link> */}
            <ExpenseListFilter />
            <h4>List of Expenses</h4>
            <ExpenseList />
        </div>
    )
}

export default Dashboard;