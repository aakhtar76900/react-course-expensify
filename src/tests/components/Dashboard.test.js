import React from 'react';
import Dashboard from '../../components/Dashboard';
import { shallow } from 'enzyme';

test('should load dashboard page' , () => {
    const wrapper = shallow(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
});