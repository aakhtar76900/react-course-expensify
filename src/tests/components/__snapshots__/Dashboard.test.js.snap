// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should load dashboard page 1`] = `
<div>
  <h3>
    Expenses Dashboard
  </h3>
  <Connect(ExpenseListFilter) />
  <h4>
    List of Expenses
  </h4>
  <Connect(ExpenseList) />
</div>
`;
